File: Client/HeThongHoTroVaQuanLyPhongKham/src/app/interfaces/bao-cao/DoanhThuReportDTO.ts
import { IHoaDon } from "../hoa-don/IHoaDon";

export interface DoanhThuReportDTO {
    tongDoanhThu: number;
    soHoaDon: number;
    trangThaiThanhToan: string;
    tuNgay: Date;
    denNgay: Date;
    danhSachHoaDon: IHoaDon[];
  }

File: Server/HeThongHoTroVaQuanLyPhongKham/Dtos/DoanhThuReportDTO.cs
namespace HeThongHoTroVaQuanLyPhongKham.Dtos
{
    public class DoanhThuReportDTO
    {
        public decimal TongDoanhThu { get; set; }
        public int SoHoaDon { get; set; }
        public string TrangThaiThanhToan { get; set; }
        public DateTime TuNgay { get; set; }
        public DateTime DenNgay { get; set; }
        public List<HoaDonDTO> DanhSachHoaDon { get; set; } = new List<HoaDonDTO>();
    }
}
